onnxruntime/core/providers/nnapi/nnapi_builtin/builders/model_builder.cc

```diff
@@ -135,7 +135,7 @@ void ModelBuilder::PreprocessActivations() {
       activation_nodes_.emplace(node->Index(), ANEURALNETWORKS_FUSED_RELU);
     } else if (op_type == "Clip") {  // Relu1 or Relu6
       float min, max;
-      if (!GetClipMinMax(GetInitializerTensors(), *node, min, max))
+      if (!GetClipMinMax(GetInitializerTensors(), *node, min, max, logging::LoggingManager::DefaultLogger()))
         continue;
```

Comment:
We have the session logger available in the EP so why are we using the default logger instead of passing the session logger through?

Reply:
The entire NNAPI EP is still using LOGS_DEFAULT instead of the session logger of the EP, using default logger for now to be consistent.
Will update the NNAPI EP to use session logger of the EP